src/ox_modules/module-email.js

```diff
@@ -68,7 +68,7 @@ module.exports = function() {
      * @param {Number} sinceMinutes - Search for emails received since the specified amount of minutes into past.
      * @param {String|Regex} subject - Return email matching the specified subject.
      * @param {Number} timeout - Timeout (in milliseconds) for waiting for the message to arrive.
-     * @return {Object} Email body and TO, FROM, SUBJECT, DATE headers.
+     * @return {Object} Email body(possible keys: html,text,textAsHtml) and TO, FROM, SUBJECT, DATE headers.
```

Comment:
For backward compatibility leave `body` as is.  It should contain `text` version.
And add 3 additional properties: `html` and `textAsHtml`